File: MasterErp.Angular/src/app/components/Inventory/models/itemCategory.ts
import { CreatorModel } from "../../Shared/models/CreatorModel";

export interface ItemCategoryModel extends CreatorModel {
    itemCategoryId: number | null;
    nameAR: string;
    nameEN: string;
    operationAccountId: number | null;
    operationAccountName: string;
    managementAccountId: number | null;
    managementAccountName: string;
    description: string;
    isActive: boolean;
    displayOrder: number | null;
}

File: MasterErp.Entities/Models/Inventory/ItemCategory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MasterErp.Entities.Models.Inventory
{
    public class ItemCategory : CreatorModel
    {
        public int ItemCategoryId { get; set; }
        public string NameAR { get; set; }
        public string NameEN { get; set; }
        public int OperationAccountId { get; set; }
        public int ManagementAccountId { get; set; }
        public string Description { get; set; }
        public bool IsActive { get; set; }
        public int? DisplayOrder { get; set; }
    }
}
